python/servo/platform/base.py

```diff
@@ -82,8 +82,9 @@ def install_cargo_deny(self, force: bool) -> bool:
         if not force and shutil.which("cargo-deny") is not None:
```

Comment:
We should also check the version here if we want to ensure >= 0.16.3 is installed. 
For me ./mach test-tidy did not complain or fail, although I still have 0.14 installed

Reply:
Okay. I've made this check. One still needs to run `./mach bootstrap` again though.